vllm/model_executor/model_loader/loader.py

```diff
@@ -883,11 +883,11 @@ def _load_weights(self, model_config: ModelConfig,
         if not hasattr(model, 'load_weights'):
             raise AttributeError(
                 "The required method 'load_weights' is not defined in class"
-                f" {type(self).__name__}.")
+                f" {type(model).__name__}.")
 
         if not hasattr(model, 'bitsandbytes_stacked_params_mapping'):
             raise AttributeError(
-                f"Model {type(self).__name__} does not support BitsAndBytes "
+                f"Model {type(model).__name__} does not support BitsAndBytes "
```

Comment:
Could we mention something specific to the stacked params mapping?

Reply:
IMHO, The meaning of `bitsandbytes_stacked_params_mapping` is not very intuitive, so I  do not recommend providing related information.